IBX-6717: Recent activity (#2161)
ibexa/documentation-developer: docs/search/activity_log_search_reference/object_name_criterion.md

# Object Name Criterion

The `ObjectNameCriterion` Activity Log Criterion

> matches log groups having a log entry with an object having a given string as name, or part of their name.

Reviewer: ```suggestion
matches log groups that have a log entry with an object having a given string as name, or part of their name.
```

## Arguments

> - `query` - The string searched for.

Reviewer 2: Please verify if correct:
```suggestion
- `query` - string(s) representing the object name
```

> - `operator` - How to compare the string to the log entry object names.

Author: ```suggestion
- `operator` - constant representing how to compare log names with the query
```
